mmv1/templates/terraform/encoders/compute_backend_bucket.go.erb

```diff
@@ -0,0 +1,36 @@
+c, cdnPolicyOk := d.GetOk("cdn_policy")
+
+if !cdnPolicyOk {
+  return obj, nil
+}
+
+cdnPolicies := c.([]interface{})
+
+if len(cdnPolicies) == 0 {
+  return obj, nil
+}
+
+cdnPolicy := cdnPolicies[0].(map[string]interface{})
+
+cacheMode := cdnPolicy["cache_mode"].(string)
+_, defaultTTLOk := cdnPolicy["default_ttl"]
+_, maxTTLOk := cdnPolicy["max_ttl"]
```

Comment:
Hi 👋  thanks for your PR! Currently the checks on this PR are failing because this variable isn't used elsewhere in the code. Does the switch statement need to be updated to use it, or can it be deleted?

Reply:
Oh you're right I forgot to clean up those variables will do !

Reply:
Done sorry for that !